contracts/access/AccessControlDefaultAdminRules.sol

```diff
@@ -53,6 +53,7 @@ abstract contract AccessControlDefaultAdminRules is IAccessControlDefaultAdminRu
      * @dev Sets the initial values for {defaultAdminDelay} and {defaultAdmin} address.
      */
     constructor(uint48 initialDelay, address initialDefaultAdmin) {
+        require(initialDefaultAdmin != address(0), "AccessControl: initial admin should not be 0");
```

Comment:
```suggestion
        require(initialDefaultAdmin != address(0), "AccessControl: 0 default admin");
```

I'm not sure how it affects on construction since it doesn't do to the bytecode, but if you think this is worth it, it's a shorter version that fits in a `bytes32`